### ISSUE TITLE
API Search functionality

### ISSUE THREAD

**Issue by zaach** [Raheem-ai/seed_web#101]
Dashboard folks are curious to know how we can search by race, age, and gender of the reporter. Dashboard issue here: https://github.com/Raheem-ai/dashboard/issues/6

Perhaps related to #85

**Comment by jhankin** [comment]
Yeah, this will need to be added.  Right now you can search by tags and reactions and ratings, but not yet by those user parameters.  I'm going to try and get you guys what you need this week.  How blocked are you by this?

**Comment by zaach** [comment]
We can make progress on other parts of the dashboard in the meantime but we'll need this for a meaningful demo. @BrandonAnderson has a demo scheduled for the 29th, so if we have it this week we should have time to test it and make sure the integration works by then.

/cc @khadijah-coakley
